lib/interactWStatusBox.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+/* USAGE
+1. import the module 
+2. addStatusBox: adds a status box to the main parent class. Takes in the input parameter of a div ID
+3. addMessageInStatus: adds a message to be displayed to a particular status bar. 
+Takes in the div ID and the message to be displayed 
+
+Example usage :
+const interact = require('./interactWStatusBox.js');
+interact.addStatusBox(11);
+interact.addMessageInStatus(11, 'this is a message');
+*/
+
+let addMessage = (divID, message) => {
+  document.getElementById(divID).innerHTML = message;
+};
+
+//default value in case user doesn't provide id
+let CountID = 1;
```

Comment:
my preference would be to eliminate the default ID mechanism because that forces the developers (us!) to give them good names, which makes for cleaner code.  Plus if one did create a status box without a name here, how would one know what the ID was? :)